content/en/blog/2025/protect-signing-secrets-for-tekton-chains-using-confidential-computing/index.md

```diff
@@ -0,0 +1,386 @@
+---
+title: Protect Signing Secrets for Tekton Chains using Confidential Computing
+linkTitle: Protect Signing Secrets for Tekton Chains using Confidential Computing
+date: 2025-04-23
+author: "Puneet Punamiya and Pradipta Banerjee, Red Hat"
+description: >
+  Integrating Tekton pipelines and chains with confidential computing
+---
+
+
+## What are Tekton Chains?
+
+[Tekton chains](https://github.com/tektoncd/chains/blob/main/README.md), a part of the Tekton project, provides tools to generate, store, and sign provenance for artifacts built with Tekton Pipelines.
+
+Technically it's a Kubernetes Custom Resource Definition (CRD) controller that allows you to manage your supply chain security in Tekton.  Chains observes Tekton Pipelines and generates provenance for the artifacts built the Pipelines.
+
+
+The following diagram from the Tekton Chains [tutorial](https://github.com/tektoncd/chains/blob/main/docs/tutorials/getting-started-tutorial.md) shows the Tekton Chains workflow
+
+![alt_text](image1.png "Speed Up Container Image Builds in Tekton Pipelines")
```

Comment:
Can we provide actual alt text for the images instead of the repeated literal "alt_text"? Otherwise the content of the images is not accessible to many users with visual impairments.